crates/wasm-compose/src/graph.rs

```diff
@@ -1554,4 +1558,344 @@ mod test {
 
         Ok(())
     }
+
+    #[test]
+    fn it_encodes_a_simple_composition_with_locked_dep() -> Result<()> {
```

Comment:
Would it be possible to move these tests into `tests/compositions`? Otherwise it'll be pretty difficult to update these test expectations if they start failing in the future

Reply:
Went ahead and changed the original tests in `tests/compositions` to include `invalid-semver` in their name, and created new ones there that capture the spirit of these, called `locked-dep-simple` and `unlocked-dep-simple`.